extensions/wikia/PortableInfobox/services/PortableInfoboxBuilderService.class.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+class PortableInfoboxBuilderService extends WikiaService {
+
+	public function translate( $builderData ) {
```

Comment:
Since it's a public method and builderData comes as JSON, please describe the format; I know it's in the tests, if you think just adding a @see PortableInfoboxBuilderServiceTest:: translationsDataProvider it's also fine with me